nex/storage-manager/activate_with_card_id.go

```diff
@@ -0,0 +1,41 @@
+package nex_storage_manager
+
+import (
+	"github.com/PretendoNetwork/mario-kart-7/globals"
+	nex "github.com/PretendoNetwork/nex-go"
+	storage_manager "github.com/PretendoNetwork/nex-protocols-go/storage-manager"
+)
+
+func ActivateWithCardID(err error, client *nex.Client, callID uint32, unknown uint8, cardID uint64) uint32 {
+	if err != nil {
+		globals.Logger.Error(err.Error())
+		return nex.Errors.Core.InvalidArgument
+	}
+
+	rmcResponseStream := nex.NewStreamOut(globals.SecureServer)
+
+	rmcResponseStream.WriteUInt32LE(1) // Unique ID
+	rmcResponseStream.WriteBool(false) // First time
```

Comment:
Is the `cardID` parameter supposed to be used here? This feels like something we should store in the Postgres DB; the users PID, `cardID`, and other data, and then handle this properly?

Reply:
No, this is hardcoded. I will fix it